TestControllers/TestController.psm1

```diff
@@ -293,7 +297,7 @@ Class TestController
 
 			# Check whether the case if for Windows images
 			$IsWindowsImage = $false
-			if($test.Tags -and $test.Tags.ToString().Contains("nested-hyperv")) {
+			if(($test.Tags -and $test.Tags.ToString().Contains("nested-hyperv")) -Or ($test.AdditionalHWConfig.OSType -contains "Windows")) {
```

Comment:
could you remove the condition "($test.Tags -and $test.Tags.ToString().Contains("nested-hyperv"))" and update the nested-hyperv cases to add <AdditionalHWConfig><OSType>Windows</OSType><AdditionalHWConfig>?

Reply:
Removed the nested-hyperv condition check and updated the NestedVmTests.xml with Windows OSType